presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+file that contains the general Hive connector configuration.
+
+Hive Metastore Thrift Service
+-----------------------------
+
+In a Kerberized Hadoop cluster, Presto accesses the Hive metastore Thrift
+service through :abbr:`SASL (Simple Authentication and Security Layer)`.
+Kerberos authentication for the metastore is configured in the connector's
+properties file using the following properties:
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.metastore.authentication.type``             One of ``SIMPLE`` or ``SASL``. When set to ``SASL`` the Hive ``SIMPLE``
+                                                   connector will connect to the Hive metastore Thrift service
+                                                   using SASL and authenticate with Kerberos. When using the
+                                                   default value of ``SIMPLE``, Kerberos authentication is
+                                                   disabled and no other properties need to be configured.
+
+``hive.metastore.principal``                       The Kerberos principal of the Hive metastore. The Presto
+                                                   coordinator will use this to authenticate the Hive
+                                                   metastore.
+
+``hive.metastore.presto.principal``                The Kerberos principal that Presto will use when connecting
+                                                   to the Hive metastore.
+
+``hive.metastore.presto.keytab``                   The path to the keytab file that contains a key for the
+                                                   principal specified by ``hive.metastore.presto.principal``.
+
+================================================== ============================================================ ==========
+
+``SIMPLE`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SIMPLE
+
+``SIMPLE`` authentication is the default authentication mechanism for the Hive
+metastore. When using ``SIMPLE`` authentication, Presto connects to an
+unsecured Hive metastore. Kerberos is not used.
+
+``SASL`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SASL
+    hive.metastore.principal=hive/hive-metastore-host.example.com@EXAMPLE.COM
+    hive.metastore.presto.principal=presto@EXAMPLE.COM
+    hive.metastore.presto.keytab=/etc/presto/hive.keytab
+
+When using ``SASL`` authentication for the Hive metastore Thrift service,
+Presto will connect as the principal specified by the property
```

Comment:
...as the Kerberos principal specified....?
(Since Kerberos is mentioned in the Simple section, and not in the SASL section.)